### PR TITLE
doc cleanup

### PR DESCRIPTION
**Pull request by mystiberry-okta**
## Description:
- Doc improvements for GA.

## Type of Pull Request:

- [x] Content (Documentation updates or typo-fixes)
- [ ] Blog Post
- [ ] Functional (CSS changes, JS updates, or layout modifications)

### Resolves:
* [OKTA-129252](https://oktainc.atlassian.net/browse/OKTA-129252)

## How Has This Been Tested?
- [x] I have built this locally and verified that it does not break existing functionality.
- [ ] For functional changes, I have tested against Chrome, Firefox, and Safari, and mobile.
- [ ] For functional changes, I have added tests.

## Primary Reviewers:
@larsjohansen-okta

### CODE REVIEW

**Comment by mystiberry-okta** [comment]
Please be aware, this is a tightly scoped clean up. There are still a few color issues to follow up on later.

File: _source/_docs/api/resources/oauth2.md [hunk 1]
```diff
@@ -2228,17 +2228,17 @@ Token limits:
...
+| id                   | ID of the claim                                                                                    | String                                    | True except for create or get all claims |
+| name                 | Name of the claim                                                                                  | String                                    | True                                     |
+| status               | Specifies whether requests have access to this claim. Valid values: `ACTIVE` or `INACTIVE`         | Enum                                      | True                                     |
+| claimType            | Specifies whether the claim is an Access Token (`RESOURCE`) or ID Token (`IDENTITY`)               | Enum                                      | True                                     |
+| valueType            | Specifies whether the claim is an Okta EL expression (`EXPRESSION`) or a set of groups (`SYSTEM`)  | Enum                                      | True                                     |
```
Comment (larsjohansen-okta) [inline _source/_docs/api/resources/oauth2.md]: ```Specifies whether the claim is an Okta EL expression (`EXPRESSION`), a set of groups (`GROUPS`) or a system claim (`SYSTEM`)```

File: _source/_docs/api/resources/oauth2.md [hunk 2]
```diff
@@ -2228,17 +2228,17 @@ Token limits:
...
+| value                | The value of the claim. Can be a string literal or an Okta EL expression.                          | String                                    | True                                     |
```
Comment (larsjohansen-okta) [inline _source/_docs/api/resources/oauth2.md]: This is not very clear; it has to be a valid Okta EL expression if the valueType is EXPRESSION and it can only be a string literal if the valueType is GROUPS. The string literal will then be matched with whatever groupFilterType is selected.

File: _source/_docs/api/resources/oauth2.md [hunk 3]
```diff
@@ -2228,17 +2228,17 @@ Token limits:
...
+|:---------------------|:---------------------------------------------------------------------------------------------------|:------------------------------------------|:-----------------------------------------|
+| id                   | ID of the claim                                                                                    | String                                    | True except for create or get all claims |
+| name                 | Name of the claim                                                                                  | String                                    | True                                     |
+| status               | Specifies whether requests have access to this claim. Valid values: `ACTIVE` or `INACTIVE`         | Enum                                      | True                                     |
+| claimType            | Specifies whether the claim is an Access Token (`RESOURCE`) or ID Token (`IDENTITY`)               | Enum                                      | True                                     |
```
Comment (larsjohansen-okta) [inline _source/_docs/api/resources/oauth2.md]: Specifies whether the claim is an Access Token -> Specifies whether the claim is for an Access Token

File: _source/_docs/api/resources/oauth2.md (hunk 2 above)
Comment (larsjohansen-okta) [inline _source/_docs/api/resources/oauth2.md]: It seems like we are missing groupFilterType in this table.

Comment (mystiberry-okta) [inline _source/_docs/api/resources/oauth2.md]: It's not anywhere in the documentation. Trying to figure out when we added it to find out what went wrong.

File: _source/_docs/api/resources/oauth2.md [hunk 4]
```diff
@@ -360,7 +360,7 @@ curl -X POST \
 ##### Response Example (Error)
 {:.api .api-response .api-response-example}
 
-~~~http
+~~~sh
```
Comment (larsjohansen-okta) [inline _source/_docs/api/resources/oauth2.md]: Why this change?

Comment (mystiberry-okta) [inline _source/_docs/api/resources/oauth2.md]: That's what fixed the color coding error, I think.
